packages/paste-core/components/sidebar/src/Sidebar.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {useSpring, animated} from '@twilio-paste/animation-library';
+import {useTheme} from '@twilio-paste/theme';
+import {useWindowSize} from '@twilio-paste/utils';
+
+import type {Variants} from './types';
+
+const StyledSidebar = React.forwardRef<HTMLDivElement, BoxProps>((props, ref) => (
+  <Box
+    aria-label="Main Navigation"
```

Comment:
issue: hard coded string for i18n

Reply:
So `'aria-label'?: string;` is a prop on the Sidebar which gets passed down to this to overwrite it. Should I remove the default label?

Reply:
Depends on if it's a mandatory prop. I can imagine people not realizing they need to supply a label and not overriding it. Even if they internationalise it, they won't know necessarily that we buried a hard coded string in the component for them to override.